processing file asmFiles/count_days.asm (1 of 12)
processing file asmFiles/count_days.asm (1 of 12)
processing file asmFiles/fib.asm (2 of 12)
processing file asmFiles/mergesort.asm (3 of 12)
--- memsim.hex	2016-09-28 18:25:01.092804171 -0400
+++ memcpu.hex	2016-09-28 18:25:01.079804686 -0400
@@ -77,133 +77,101 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
+:0400C1000000001526
+:0400C2000000001624
+:0400C3000000001920
+:0400C400000000290F
+:0400C5000000003304
+:0400C60000000050E6
+:0400C70000000051E4
+:0400C8000000005ADA
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:0401400000000015A6
+:0401410000000016A4
+:0401420000000019A0
+:04014300000000298F
+:040144000000003384
+:040145000000005066
+:040146000000005164
+:040147000000005A5A
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
 :040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 12)
processing file asmFiles/mult_procedure.asm (5 of 12)
processing file asmFiles/search.asm (6 of 12)
processing file asmFiles/test.asm (7 of 12)
processing file asmFiles/test.branchjump1.asm (8 of 12)
processing file asmFiles/test.ldst1.asm (9 of 12)
processing file asmFiles/test.ldst2.asm (10 of 12)
processing file asmFiles/test.loadstore.asm (11 of 12)
processing file asmFiles/test.rtype.asm (12 of 12)
processing file asmFiles/count_days.asm (1 of 12)
processing file asmFiles/fib.asm (2 of 12)
processing file asmFiles/mergesort.asm (3 of 12)
--- memsim.hex	2016-09-28 19:57:59.161035386 -0400
+++ memcpu.hex	2016-09-28 19:57:59.145036018 -0400
@@ -77,133 +77,101 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
+:0400C1000000001526
+:0400C2000000001624
+:0400C3000000001920
+:0400C400000000290F
+:0400C5000000003304
+:0400C60000000050E6
+:0400C70000000051E4
+:0400C8000000005ADA
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:0401400000000015A6
+:0401410000000016A4
+:0401420000000019A0
+:04014300000000298F
+:040144000000003384
+:040145000000005066
+:040146000000005164
+:040147000000005A5A
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
 :040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 12)
processing file asmFiles/mult_procedure.asm (5 of 12)
processing file asmFiles/search.asm (6 of 12)
processing file asmFiles/test.asm (7 of 12)
processing file asmFiles/test.branchjump1.asm (8 of 12)
processing file asmFiles/test.ldst1.asm (9 of 12)
processing file asmFiles/test.ldst2.asm (10 of 12)
processing file asmFiles/test.loadstore.asm (11 of 12)
processing file asmFiles/test.rtype.asm (12 of 12)
processing file asmFiles/count_days.asm (1 of 12)
--- memsim.hex	2016-09-30 19:55:26.987528695 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,49 +1,39 @@
-:04000000341DFFFCB0
-:04000100340B0008B4
-:04000200340C0013A7
-:04000300340D07E0D1
-:04000400341CFFF8B1
-:040005000180502105
-:0400060034070001BA
-:040007000167582312
-:040008003407001E9B
-:0400090027BDFFFC14
-:04000A00AFA700009C
-:04000B0027BDFFFC12
-:04000C00AFAB000096
-:04000D000C00001DC6
-:04000E008FAB0000B4
-:04000F0027BD000405
-:04001000014B50212F
-:04001100340707D0D9
-:0400120001A76823B7
-:040013003407016D40
-:0400140027BDFFFC09
-:04001500AFA7000091
-:0400160027BDFFFC07
-:04001700AFAD000089
-:040018000C00001DBB
-:040019008FAD0000A7
-:04001A0027BD0004FA
-:04001B00014D502122
-:04001C000800002BAD
-:04001D008FA20000AE
-:04001E0027BD0004F6
-:04001F008FA30000AB
-:0400200027BD0004F4
-:04002100006020213A
-:0400220034050000A1
-:04002300340600019E
-:040024001080000345
-:0400250000A22821EC
-:04002600008620230D
-:0400270008000024A9
-:0400280027BDFFFCF5
-:04002900AFA500007F
-:04002A0003E00008E7
-:04002B0027BDFFFCF2
-:04002C00AFAA000077
-:04002D00FFFFFFFFD3
-:043FFD0000000010B0
-:043FFE00000017B5F3
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/fib.asm (2 of 12)
--- memsim.hex	2016-09-30 19:55:27.116523593 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,38 +1,39 @@
-:040000003442008006
-:0400010034210001A5
-:0400020035080004B9
-:0400030034C60F00F0
-:040004008CCF00009D
-:040005008C43000028
-:040006008C44000422
-:040007000064282148
-:04000800AC450008FB
-:04000900004810217A
-:04000A0001E1782375
-:04000B0015E0FFF904
-:04000C00FFFFFFFFF4
-:0400210000000001DA
-:0400220000000001D9
-:0400230000000002D7
-:0400240000000003D5
-:0400250000000005D2
-:0400260000000008CE
-:040027000000000DC8
-:0400280000000015BF
-:0400290000000022B1
-:04002A00000000379B
-:04002B000000005978
-:04002C000000009040
-:04002D00000000E9E6
-:04002E000000017954
-:04002F000000026269
-:04003000000003DBEE
-:040031000000063D88
-:0400320000000A18A8
-:040033000000105564
-:0400340000001A6D41
-:0400350000002AC2DB
-:040036000000452F52
-:0400370000006FF165
-:0403C0000000001623
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/mergesort.asm (3 of 12)
--- memsim.hex	2016-09-30 19:55:27.225519281 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,209 +1,39 @@
-:04000000341EFFFCAF
-:04000100341DFFFCAF
-:0400020034040304BB
-:040003008C1003005A
-:04000400020028428C
-:040005000004882546
-:04000600000590253C
-:040007000C00001ACF
-:040008000200404270
-:04000900020828239E
-:04000A000100408031
-:04000B0034040304B2
-:04000C000088202127
-:04000D00000498252E
-:04000E000005A02524
-:04000F000C00001AC7
-:040010000011202596
-:04001100001228258C
-:040012000013302582
-:040013000014382578
-:0400140034080500A7
-:0400150027BDFFFC08
-:04001600AFA800008F
-:040017000C00002BAE
-:0400180027BD0004FC
-:04001900FFFFFFFFE7
-:04001A0034080004A2
-:04001B0000A0488079
-:04001C000109082BA3
-:04001D001020000CA3
-:04001E000088C8216D
-:04001F008F38000016
-:04002000132400069F
-:040021008F2FFFFC22
-:04002200030F082A96
-:0400230010200003A6
-:04002400AF2F0000FA
-:040025002739FFFC7C
-:0400260008000020AE
-:04002700AF380000EE
-:0400280025080004A3
-:040029000800001CAF
-:04002A0003E00008E7
-:04002B008FA800009A
-:04002C0014A000021A
-:04002D0014E00001DA
-:04002E000800004D79
-:04002F0010E0000FCE
-:0400300010A0001507
-:040031008C890000B6
-:040032008CCA000074
-:04003300012A082A6C
-:040034001020000593
-:04003500AD09000011
-:040036002508000495
-:040037002484000419
-:0400380024A5FFFFFD
-:040039000800002C8F
-:04003A00AD0A00000B
-:04003B002508000490
-:04003C0024C60004D2
-:04003D0024E7FFFFB6
-:04003E000800002C8A
-:04003F008C890000A8
-:04004000AD09000006
-:04004100250800048A
-:0400420024A5FFFFF3
-:04004300248400040D
-:0400440010A0000800
-:040045000800003F70
-:040046008CCA000060
-:04004700AD0A0000FE
-:040048002508000483
-:0400490024E7FFFFAA
-:04004A0024C60004C4
-:04004B0010E00001C0
-:04004C000800004662
-:04004D0003E00008C4
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
-:043FFE0000000500BA
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/mult.asm (4 of 12)
--- memsim.hex	2016-09-30 19:55:27.332515047 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,24 +1,39 @@
-:04000000341DFFFCB0
-:04000100340B0002BA
-:04000200340C0003B7
-:0400030027BDFFFC1A
-:04000400AFAB00009E
-:0400050027BDFFFC18
-:04000600AFAC00009B
-:040007008FA20000C4
-:0400080027BD00040C
-:040009008FA30000C1
-:04000A0027BD00040A
-:04000B000060202150
-:04000C0034050000B7
-:04000D0034060001B4
-:04000E00108000035B
-:04000F0000A2282102
-:040010000086202323
-:040011000800000ED5
-:0400120027BDFFFC0B
-:04001300AFA5000095
-:04001400FFFFFFFFEC
-:043FFD0000000003BD
-:043FFE0000000006B9
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/mult_procedure.asm (5 of 12)
--- memsim.hex	2016-09-30 19:55:27.442510696 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,41 +1,39 @@
-:04000000341DFFFCB0
-:04000100340B00D2EA
-:04000200340C0002B8
-:04000300340D00BAFE
-:04000400340E0008AE
-:04000500341CFFF8B0
-:0400060027BDFFFC17
-:04000700AFAB00009B
-:0400080027BDFFFC15
-:04000900AFAC000098
-:04000A0027BDFFFC13
-:04000B00AFAD000095
-:04000C0027BDFFFC11
-:04000D00AFAE000092
-:04000E0013BC00100F
-:04000F000C000011D0
-:040010000800000ED6
-:040011008FA20000BA
-:0400120027BD000402
-:040013008FA30000B7
-:0400140027BD000400
-:040015000060202146
-:0400160034050000AD
-:0400170034060001AA
-:040018001080000351
-:0400190000A22821F8
-:04001A000086202319
-:04001B0008000018C1
-:04001C0027BDFFFC01
-:04001D00AFA500008B
-:04001E0003E00008F3
-:04001F008FAA0000A4
-:0400200027BD0004F4
-:0400210027BDFFFCFC
-:04002200AFAA000081
-:04002300FFFFFFFFDD
-:043FFB0000000008BA
-:043FFC00000005D0EC
-:043FFD0000000BA015
-:043FFE0000098940ED
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/search.asm (6 of 12)
--- memsim.hex	2016-09-30 19:55:27.594504683 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,120 +1,39 @@
-:04000000341D00802B
-:0400010034010001C5
-:0400020034020004C0
-:04000300AFA00000AA
-:040004008FA30004C2
-:040005008FA40008BC
-:0400060027A5000C1E
-:040007008CA60000C3
-:0400080000C33823D6
-:0400090010E00004FF
-:04000A0000A2282107
-:04000B00008120232D
-:04000C00108000025E
-:04000D001000FFF9E7
-:04000E00AFA500009A
-:04000F00FFFFFFFFF1
-:0400200000000124B7
-:0400210000005C6F10
-:040022000000006476
-:040023000000087D54
-:0400240000005FCBAE
-:040025000000A41A19
-:04002600000041098C
-:04002700000045226E
-:040028000000700F55
-:040029000000766DF0
-:04002A0000006F6003
-:04002B0000008A5EE9
-:04002C0000009580BB
-:04002D00000070A3BC
-:04002E000000AEA977
-:04002F000000711A42
-:0400300000006F81DC
-:0400310000008F9AA2
-:040032000000258421
-:040033000000A5998B
-:040034000000401573
-:040035000000CE8178
-:040036000000F55B76
-:040037000000399EEE
-:040038000000A23FE3
-:040039000000358806
-:04003A00000033ACE3
-:04003B000000BCE71E
-:04003C0000002A6B2B
-:04003D0000009FA17F
-:04003E000000C94BAA
-:04003F000000C65B9C
-:040040000000006854
-:040041000000F4992E
-:0400420000005F71EA
-:040043000000D06F7A
-:04004400000014DFC5
-:040045000000116541
-:040046000000F88D31
-:0400470000004BA4C6
-:0400480000002E7412
-:0400490000005C6FE8
-:04004A000000D11EC3
-:04004B0000009222FD
-:04004C000000ACDB29
-:04004D000000103867
-:04004E000000AB17EC
-:04004F000000F7CEE8
-:0400500000008A9E84
-:0400510000009AA36E
-:040052000000B49561
-:0400530000008A5EC1
-:040054000000D85977
-:0400550000000BACF0
-:040056000000D0DBFB
-:04005700000035521E
-:040058000000A6B04E
-:040059000000727FB2
-:04005A00000028E496
-:04005B000000E5CFED
-:04005C000000163C4E
-:04005D00000034115A
-:04005E0000008F0708
-:04005F000000FAB7EC
-:0400600000000F3459
-:040061000000DABF02
-:0400620000006F6FBC
-:040063000000C5983C
-:040064000000F4960E
-:0400650000009A9A63
-:040066000000BD6A6F
-:04006700000021363E
-:040068000000810A09
-:040069000000CA5574
-:04006A0000008BCE39
-:04006B0000002AC4A3
-:04006C000000DDCEE5
-:04006D000000DD06AC
-:04006E000000C4FCCE
-:04006F000000FB2F63
-:040070000000EE5F3F
-:040071000000FD305E
-:040072000000C54085
-:040073000000D5F1C3
-:040074000000BDAD1E
-:04007500000045C37F
-:040076000000708A8C
-:040077000000A35989
-:040078000000F40D83
-:040079000000BA06C3
-:04007A000000BACEFA
-:04007B000000B44786
-:04007C0000003F48F9
-:04007D000000899E58
-:04007E00000080847A
-:04007F000000BDB907
-:040080000000A05A82
-:040081000000E22574
-:040082000000FB0C73
-:040083000000B2B215
-:040084000000A4DBF9
-:0400850000008BF9F3
-:04008600000012F76D
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.asm (7 of 12)
--- memsim.hex	2016-09-30 19:55:27.702500411 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,21 +1,39 @@
-:04000000341DFFFCB0
-:04000100340B00BC00
-:04000200340C0014A6
-:0400030027BDFFFC1A
-:04000400AFAB00009E
-:0400050027BDFFFC18
-:04000600AFAC00009B
-:040007008FA20000C4
-:0400080027BD00040C
-:040009008FA30000C1
-:04000A0027BD00040A
-:04000B00340F0800A6
-:04000C0034100804A0
-:04000D00ADE2000060
-:04000E00AE0300003D
-:04000F00FFFFFFFFF1
-:0402000000000014E6
-:04020100000000BC3D
-:043FFD0000000014AC
-:043FFE00000000BC03
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.branchjump1.asm (8 of 12)
--- memsim.hex	2016-09-30 19:55:27.812496059 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,18 +1,39 @@
-:040000003401BA5CB1
-:040001003402008045
-:04000200340F003483
-:0400030010000001E8
-:04000400AC4100000B
-:040005000C000009E2
-:04000600AC41000405
-:04000700AC5F0010DA
-:04000800FFFFFFFFF8
-:04000900001F182597
-:04000A00AC5F0008DF
-:04000B000C00000DD8
-:04000C00AC41000CF7
-:04000D0017E3FFF9FD
-:04000E00FFFFFFFFF2
-:0400220000000018C2
-:0400240000000030A8
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.ldst1.asm (9 of 12)
--- memsim.hex	2016-09-30 19:55:27.920491787 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,21 +1,39 @@
-:04000000340100F0D7
-:0400010034020100C4
-:0400020034030200C1
-:0400030034040300BE
-:0400040034050400BB
-:040005008C26000045
-:040006008C2700043F
-:040007008C28000839
-:0400080034040500B7
-:0400090034050600B4
-:04000A00AC46000000
-:04000B00AC470004FA
-:04000C00AC480008F4
-:04000D00FFFFFFFFF3
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.ldst2.asm (10 of 12)
--- memsim.hex	2016-09-30 19:55:28.029487474 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,20 +1,39 @@
-:040000003C0AFEEDCB
-:0400010034010F00B7
-:0400020034020800BC
-:04000300354ABEEFCD
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC4A000CF0
-:04000B00FFFFFFFFF5
-:040200000000733750
-:0402010000002701D1
-:0402020000001337AE
-:04020300FEEDBEEF5F
-:0403C000000073378F
-:0403C1000000270110
-:0403C20000001337ED
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.loadstore.asm (11 of 12)
--- memsim.hex	2016-09-30 19:55:28.172481817 -0400
+++ memcpu.hex	2016-09-28 19:58:15.384393490 -0400
@@ -1,20 +1,39 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.rtype.asm (12 of 12)
processing file asmFiles/count_days.asm (1 of 12)
processing file asmFiles/fib.asm (2 of 12)
processing file asmFiles/mergesort.asm (3 of 12)
--- memsim.hex	2016-09-30 19:56:03.385088737 -0400
+++ memcpu.hex	2016-09-30 19:56:03.367089449 -0400
@@ -77,133 +77,101 @@
 :04004C000800004662
 :04004D0003E00008C4
 :0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
+:0400C1000000001526
+:0400C2000000001624
+:0400C3000000001920
+:0400C400000000290F
+:0400C5000000003304
+:0400C60000000050E6
+:0400C70000000051E4
+:0400C8000000005ADA
+:0400C9000000000C27
+:0400CA000000003EF4
+:0400CB000000004BE6
+:0400CC0000000047E9
+:0400CD0000000053DC
+:0400CE0000000051DD
+:0400CF000000004DE0
+:0400D0000000001616
+:0400D1000000000B20
+:0400D2000000001D0D
+:0400D3000000000722
+:0400D4000000002107
+:0400D50000000063C4
+:0400D6000000001B0B
+:0400D70000000064C1
+:0400D80000000042E2
+:0400D9000000003DE6
+:0400DA000000002002
+:0400DB000000000120
+:0400DC0000000036EA
+:0400DD00000000041B
+:0400DE000000003DE1
+:0400DF0000000038E5
+:0400E0000000000319
+:0400E10000000030EB
+:0400E2000000000812
+:0400E30000000042D7
+:0400E40000000064B4
+:0400E5000000000F08
+:0400E6000000005CBA
+:0400E70000000041D4
+:0400E80000000020F4
+:0400E900000000090A
+:0400EA000000002FE3
+:0400EB0000000059B8
+:0400EC0000000011FF
+:0400ED000000000708
+:0400EE0000000023EB
+:0400EF0000000044C9
+:0400F00000000020EC
+:0400F1000000000A01
+:0400F2000000000703
+:0400F30000000017F2
+:0400F4000000005CAC
+:0400F5000000005BAC
+:0400F60000000028DE
+:0400F7000000001AEB
+:0400F80000000008FC
+:0400F90000000024DF
+:0400FA0000000026DC
+:0400FB0000000008F9
+:0400FC0000000026DA
+:0400FD0000000010EF
+:0400FE0000000032CC
+:0400FF0000000007F6
+:0401000000000043B8
+:0401400000000015A6
+:0401410000000016A4
+:0401420000000019A0
+:04014300000000298F
+:040144000000003384
+:040145000000005066
+:040146000000005164
+:040147000000005A5A
+:040148000000000CA7
+:040149000000003E74
+:04014A000000004B66
+:04014B000000004769
+:04014C00000000535C
+:04014D00000000515D
+:04014E000000004D60
+:04014F000000001696
+:040150000000000BA0
+:040151000000001D8D
+:0401520000000007A2
+:040153000000002187
+:040154000000006344
+:040155000000001B8B
+:040156000000006441
+:040157000000004262
+:040158000000003D66
 :040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
+:04015A0000000001A0
+:04015B00000000366A
+:04015C00000000049B
+:04015D000000003D61
+:04015E000000003865
+:04015F000000000399
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 12)
processing file asmFiles/mult_procedure.asm (5 of 12)
processing file asmFiles/search.asm (6 of 12)
processing file asmFiles/test.asm (7 of 12)
processing file asmFiles/test.branchjump1.asm (8 of 12)
processing file asmFiles/test.ldst1.asm (9 of 12)
processing file asmFiles/test.ldst2.asm (10 of 12)
processing file asmFiles/test.loadstore.asm (11 of 12)
processing file asmFiles/test.rtype.asm (12 of 12)
